web_environment_ribbon/static/src/js/ribbon.js

```diff
@@ -30,27 +31,21 @@ core.bus.on('web_client_ready', null, function () {
     var ribbon = $('<div class="test-ribbon"/>');
     $('body').append(ribbon);
     ribbon.hide();
-    model.call('get_param', ['ribbon.name']).then(
-        function (name) {
-            if (name && name != 'False') {
-                ribbon.html(name);
+    // Get ribbon data from backend
+    backend_model.call('get_environment_ribbon').then(
+        function (ribbon_data) {
+            // Ribbon name
+            if (ribbon_data.name && ribbon_data.name != 'False') {
+                ribbon.html(ribbon_data.name);
                 ribbon.show();
             }
-        }
-    );
-    // Get ribbon color from system parameters
-    model.call('get_param', ['ribbon.color']).then(
-        function (strColor) {
-            if (strColor && validStrColour(strColor)) {
-                ribbon.css('color', strColor);
+            // Ribbon color
+            if (ribbon_data.color && validStrColour(ribbon_data.color)) {
```

Comment:
@zakiuu Hi. It's the behavior before my PR. If it's an issue, we should fix it in a separate PR.